robottelo/config/__init__.py

```diff
@@ -15,21 +15,31 @@
     os.environ['ROBOTTELO_DIR'] = str(robottelo_root_dir)
 
 
-settings = LazySettings(
-    envvar_prefix="ROBOTTELO",
-    core_loaders=["YAML"],
-    settings_file="settings.yaml",
-    preload=["conf/*.yaml"],
-    includes=["settings.local.yaml", ".secrets.yaml", ".secrets_*.yaml"],
-    envless_mode=True,
-    lowercase_read=True,
-)
-settings.validators.register(**VALIDATORS)
-
-try:
-    settings.validators.validate()
-except ValidationError as err:
-    logger.warning(f'Dynaconf validation failed, continuing for the sake of unit tests\n{err}')
+def get_settings():
+    """Return Lazy settings object after validating
+
+    :return: A validated Lazy settings object
+    """
+    settings = LazySettings(
+        envvar_prefix="ROBOTTELO",
+        core_loaders=["YAML"],
+        settings_file="settings.yaml",
+        preload=["conf/*.yaml"],
+        includes=["settings.local.yaml", ".secrets.yaml", ".secrets_*.yaml"],
+        envless_mode=True,
+        lowercase_read=True,
+    )
+    settings.validators.register(**VALIDATORS)
+
+    try:
+        settings.validators.validate()
+    except ValidationError as err:
+        logger.warning(f'Dynaconf validation failed, continuing for the sake of unit tests\n{err}')
```

Comment:
validation shouldn't happen in the get_settings function. it is a step someone may perform after.

Reply:
Since we're currently ignoring all validation errors anyway, why should this method not return validated settings? I'd expect this function to return a validated, usable settings object, not a settings object that I then have to import validators for, or call methods on to check it before I use it.

Reply:
@JacobCallahan What would be an example of needing to perform settings validation after? I can always move the try/except block outside of get_settings